pycqed/measurement/detector_functions.py

```diff
@@ -581,6 +596,41 @@ def poll_data(self):
                                 self.UHF_map[UHFname]].nr_sweep_points:
                                 gotem[UHFname][n] = True
             accumulated_time += 0.01 * len(self.UHFs)
+            if print_progress:
+                t_now = time.time()
+                if t_now - t_callback > self.progress_callback_interval:
+                    try:
+                        if self.ro_mode == 'rl':
+                            n_acq = [UHF.qas_0_result_acquired() for UHF in
+                                     self.UHFs]
+                        else:
+                            n_acq = [UHF.qas_0_monitor_acquired() for UHF in
+                                     self.UHFs]
+                        # FIXME: This workaround is needed because the
```

Comment:
I think we should remove the FIXME here. As we discussed, it is something ZI must fix not us. Or make it explicit that the FIXME me here is for ZI because when I first saw it I thought this part of the code still needs work on our side.